khelo/lib/ui/flow/tournament/add/add_tournament_screen.dart

```diff
@@ -0,0 +1,275 @@
+        onEditButtonTap: () async {
+          final imagePath = await ImagePickerSheet.show<String>(context, true);
+          if (imagePath != null) {
+            notifier.onImageChange(imagePath);
+          }
+        },
+      ),
+    );
+  }
+
+  Widget _selectTileView({
+    String? label,
+    IconData? icon,
+    required String title,
+    required Function() onTap,
+  }) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        if (label != null) ...[
+          Text(
+            label,
+            style: AppTextStyle.body2.copyWith(
+              color: context.colorScheme.textDisabled,
+            ),
+          ),
+          const SizedBox(height: 12),
+        ],
+        OnTapScale(
+          onTap: onTap,
+          child: Material(
+            type: MaterialType.transparency,
+            child: ListTile(
+              dense: true,
+              contentPadding: const EdgeInsets.symmetric(horizontal: 12),
+              shape: RoundedRectangleBorder(
+                side: BorderSide(color: context.colorScheme.outline),
+                borderRadius: BorderRadius.circular(12),
+              ),
+              title: Text(
+                title,
+                style: AppTextStyle.body2.copyWith(
+                  color: context.colorScheme.textPrimary,
+                ),
+              ),
+              trailing: Icon(
+                icon,
+                color: context.colorScheme.textDisabled,
+              ),
+            ),
+          ),
+        ),
+      ],
+    );
+  }
+
+  Widget _dateScheduleView(
+    BuildContext context,
+    AddTournamentState state,
+  ) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Row(
+          children: [
+            Expanded(
+              child: AdaptiveOutlinedTile(
+                  title: state.startDate.format(context, DateFormatType.date),
+                  headerText: context.l10n.add_tournament_start_date,
+                  placeholder: context.l10n.add_tournament_start_date,
+                  onTap: () {
+                    selectDate(
+                      context,
+                      initialDate: state.startDate,
+                      onDateSelected: (date) => notifier.onStartDate(date),
+                    );
+                  }),
+            ),
+            const SizedBox(width: 16),
+            Expanded(
+              child: AdaptiveOutlinedTile(
+                  title: state.endDate.format(context, DateFormatType.date),
+                  headerText: context.l10n.add_tournament_end_date,
+                  placeholder: context.l10n.add_tournament_end_date,
+                  onTap: () {
+                    selectDate(
+                      context,
+                      initialDate: state.endDate,
+                      onDateSelected: (date) => notifier.onEndDate(date),
+                    );
```

Comment:
can we add limitation here like, end date can be greater than start date only, also check for the same while enabling button for add tournament